rero_ils/modules/documents/mappings/v6/documents/document-v0.0.1.json

```diff
@@ -121,7 +183,26 @@
           "type": "object",
           "properties": {
             "name": {
-              "type": "text"
+              "type": "text",
```

Comment:
publisher is a name thus we do not need languages analyzers.

Reply:
done